pages/activities.jsx

```diff
@@ -154,6 +154,15 @@ var ActivitiesPage = React.createClass({
               description="These six hands-on activities engage learners in thinking critically about online privacy by creating secure passwords, understanding how and where their data is being collected, and more. This is an ideal kit for Mozilla Clubs."
             />
             <ActivityKit
+              src1x="https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png"
+              src2x="https://upload.wikimedia.org/wikipedia/commons/9/98/SafeWalletLogo.png"
+              title="Privacy Basics: Passwords, Tracking, and Data Retention"
+              level="Intermediate"
+              link="/activities/privacy-basics/"
+              developedBy="Stacy Martin, Senior Data Privacy Manager at Mozilla and the Mozilla Learning Network team"
```

Comment:
this shows up as

<img width="720" alt="screen shot 2015-10-26 at 2 02 38 pm" src="https://cloud.githubusercontent.com/assets/2896608/10742403/45ca2276-7bea-11e5-913a-f85261ffd72f.png">

(notice the "and" after the comma) Which doesn't read right - Stacy *is* the Senior Data Privacy Manager at Mozilla. 

We probably need to clean up the `render` function of `ActivityKit`.  We used to **only** have comma separated string for the "DEVELOPED BY" field that's how the `authorLinks` map came in handy.  I guess that approach needs to change now before nested if-else blocks get more conplicated?  :grin: